### DISCUSSION TITLE
About Permissions

### DISCUSSION THREAD

**Discussion by hugopvl** [directus/directus#6344] [Archive]
Hey guys,

A question about permissions;

1- A user who only has viewing permissions cannot zoom in on the images? do I need to give "update" permission to zoom in?

2- The same happens with pdf files for example, with viewing permissions you can't see or download the files, do I need to give  update permission to be able to do that?

Any solutions for this situation?

Thanks Guys.

**Comment by rijkvanzanten** [comment]
> 1- A user who only has viewing permissions cannot zoom in on the images? do I need to give "update" permission to zoom in?

What do you mean by "Zoom in"? Are you talking about editing/cropping the image?

> 2- The same happens with pdf files for example, with viewing permissions you can't see or download the files, do I need to give update permission to be able to do that?

You should be able to download the file with just read permissions. Directus doesn't currently render previews for PDFs on the page though, so that might be part of the confusion here

**Reply by hugopvl** [reply]
hey guys,
no, I can't.
with read permission I can't zoom or download files,

As shown in the images

![read_permissions](https://user-images.githubusercontent.com/81326296/122559964-e9400e00-d037-11eb-920d-1ea4da453a43.png)
![read_permissions2](https://user-images.githubusercontent.com/81326296/122559968-e9d8a480-d037-11eb-98c6-a9bb7cd62132.png)

I need to give update permission to zoom and download.

![update_permissions](https://user-images.githubusercontent.com/81326296/122560023-f957ed80-d037-11eb-827b-4ccf6a307d0f.png)
![update_permissions2](https://user-images.githubusercontent.com/81326296/122560025-f9f08400-d037-11eb-8a06-542b95049a09.png)

**Reply by rijkvanzanten** [reply]
Ohh, just from the interface's perspective. Yeah, those are now just a generic "show/hide actions" when the interface is disabled. Feel free to open a PR to have some of those actions still available, even on disabled fields 👍🏻
